package.json

```diff
@@ -1,48 +1,45 @@
 {
```

Comment:
What's the reason for the re-jig of this file?

Reply:
Alphabetisation :)

Reply:
No objection - but boy will this be a fun rebase to sort!